pybaseball/split_stats.py

```diff
@@ -0,0 +1,129 @@
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from .datahelpers import postprocessing
+import bs4 as bs
+import pandas as pd
+import re
+
+
+def download_url(url: str) -> bytes:
+    """
+    Gets the content from the url specified
+    """
+    session = requests.Session()
+    retry = Retry(connect=3, backoff_factor=0.5)
+    adapter = HTTPAdapter(max_retries=retry)
+    session.mount('http://', adapter)
+    session.mount('https://', adapter)
+    resp = session.get(url)
+    return resp.content
+
+
+def get_split_soup(playerid: str, year: int = None, pitching_splits: bool = False) -> bs.BeautifulSoup:
```

Comment:
Technically an int can't be None.

To do this correctly, you have to include:

`from typing import Optional`

And then change this from `int` to `Optional[int]`